frontend/src/pages/RunList.tsx

```diff
@@ -47,14 +46,15 @@ interface RecurringRunInfo {
 interface DisplayRun {
   experiment?: ExperimentInfo;
   recurringRun?: RecurringRunInfo;
-  run: ApiRun;
+  run: V2beta1Run;
```

Comment:
We still want to show V1 Runs. Please confirm if we can still support listing V1 Runs here.